dali/pipeline/data/types_test.cc

```diff
@@ -44,14 +48,27 @@ TYPENAME_FUNC(float);
 TYPENAME_FUNC(double);
 TYPENAME_FUNC(bool);
 
+template <>
+std::string TypesTest<std::vector<uint8>>::TypeName() {
+  return "list of uint8";
+}
+
+template <>
+std::string TypesTest<std::array<std::vector<uint8>, DUMMY_ARRAY_SIZE>>::TypeName() {
+  return "list of list of uint8";
+}
+
 typedef ::testing::Types<uint8,
                          int16,
                          int32,
                          int64,
                          float16,
                          float,
                          double,
-                         bool> TestTypes;
+                         bool,
+                         std::vector<uint8>,
```

Comment:
Why we ned that?

Reply:
To have better test coverage of this part - I wanted to make sure this change does not break anything.